tests/test_gemini/update/test_optim.py

```diff
@@ -40,8 +41,8 @@ def check_param(model: ZeroDDP, torch_model: torch.nn.Module):
 
 
 # 'gpt2', 'bert',
-TEST_MODELS = ['gpt2', 'bert']
-EXAMPLE_MODELS = ['simple_net']
```

Comment:
Why remove example models? If the two exam functions test the same thing, you do not need to test twice.

Reply:
sorry, I revert it back.

Reply:
DONE. add all possible models.